{
  "uid" : "81dcf7d7135eeca2",
  "name" : "token has correct name",
  "fullName" : "Contract: ERC721PresetMinterPauserAutoId.token has correct name",
  "historyId" : "8c0de360eae492fc289869bbe6a3a2c8",
  "time" : {
    "start" : 1635782740651,
    "stop" : 1635782741688,
    "duration" : 1037
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067cbe0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067cbe0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC721/presets/ERC721PresetMinterPauserAutoId.test.js:20:58)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ERC721PresetMinterPauserAutoId"
  }, {
    "name" : "testClass",
    "value" : "Contract: ERC721PresetMinterPauserAutoId"
  }, {
    "name" : "testMethod",
    "value" : "token has correct name"
  }, {
    "name" : "package",
    "value" : "Contract: ERC721PresetMinterPauserAutoId"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "eafdd7becbbd4ac3",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17aa8e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782614743,
        "stop" : 1635782615319,
        "duration" : 576
      }
    }, {
      "uid" : "5077ebf1ebe0cb85",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0453130>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782613555,
        "stop" : 1635782614111,
        "duration" : 556
      }
    }, {
      "uid" : "13826c0a59374c0d",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0437af0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782613177,
        "stop" : 1635782613738,
        "duration" : 561
      }
    }, {
      "uid" : "4f8dea944c0dca63",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0427cd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782597012,
        "stop" : 1635782597976,
        "duration" : 964
      }
    }, {
      "uid" : "6c4622e98b22c50f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067cc10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782587448,
        "stop" : 1635782588007,
        "duration" : 559
      }
    }, {
      "uid" : "48f7e0fefeb109be",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067c280>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782534196,
        "stop" : 1635782534764,
        "duration" : 568
      }
    }, {
      "uid" : "80483e167d05dd3a",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067c4f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782522289,
        "stop" : 1635782522844,
        "duration" : 555
      }
    }, {
      "uid" : "7a591883215160d7",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0637640>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782519791,
        "stop" : 1635782520354,
        "duration" : 563
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "661fdc9d4a6e58ae",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/661fdc9d4a6e58ae",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635590440422,
          "stop" : 1635590459034,
          "duration" : 18612
        }
      }, {
        "uid" : "d787e004ecfc4d26",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/d787e004ecfc4d26",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0437cd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635555015722,
          "stop" : 1635555016709,
          "duration" : 987
        }
      }, {
        "uid" : "6cde218158c2acf8",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/6cde218158c2acf8",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551256320,
          "stop" : 1635551256399,
          "duration" : 79
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "81dcf7d7135eeca2.json",
  "parameterValues" : [ ]
}